samples/languages/python/README.md

```diff
@@ -1,20 +1,43 @@
 SGX-LKL-OE Python and NumPy Sample Application
 ==============================================
 
-1. First build the Python and NumPy file system image:
+In this sample you will run a simple Python application in SGX-LKL and get used to the available tools.
+The aim of this sample is to verify that an application works in SGX-LKL without considering more advanced topics like encryption, attestation, networking, and deployment via Docker.
 
-```
-make
+The Python application is packaged as a Docker image. Let's first verify that it works outside of SGX-LKL:
+
+```sh
+docker build -t pythonapp .
+docker run --rm pythonapp
+# Output:
+# Confidential Computing using Intel SGX in Python with NumPy... 
```

Comment:
Why the message says "using Intel SGX"? The host mode doesn't use SGX, right?

Reply:
Changed to "SGX-LKL".